airflow/jobs.py

```diff
@@ -1769,7 +1623,7 @@ def _execute_helper(self, processor_manager):
         settings.Session.remove()
 
     @provide_session
-    def process_file(self, file_path, pickle_dags=False, session=None):
+    def process_file(self, file_path, zombies, pickle_dags=False, session=None):
```

Comment:
I assume there are no other places using`process_file` as it is a breaking change?

Reply:
Yes I traced back in the code base and saw only the usage in DagFileProcessor, which is updated.